Dangerfile

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+# Warns when there is a big PR
+warn('Big PR') if git.lines_of_code > 500
```

Comment:
Where did you get the 500 lines of code number from? Not sure if it's a good number or not 😓 
As an improvement, how about being more detailed in the message e.g "This PR is too large. Prefer splitting it into smaller PRs." 💭

Reply:
This is the default command, let me remove it, we can comment directly in PR